clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -216,7 +271,10 @@ private void parseTransport(XMLExtendedStreamReader reader, PathAddress containe
                     break;
                 }
                 case EXECUTOR: {
-                    readAttribute(reader, i, operation, JGroupsTransportResourceDefinition.ExecutorAttribute.TRANSPORT);
+                    if (this.schema.since(InfinispanSchema.VERSION_3_0)) {
+                        throw ParseUtils.unexpectedAttribute(reader, i);
+                    }
+                    ROOT_LOGGER.executorIgnored(JGroupsTransportResourceDefinition.ExecutorAttribute.TRANSPORT.getDefinition().getName());
```

Comment:
This means that a domain.xml file can't contain old configs that can get sent to a legacy HC